src/surfaces/filtersDrawer/filtersDrawer.jsx

```diff
@@ -225,12 +229,14 @@ class FiltersDrawer extends React.Component {
         const { nestedTogglesData } = this.state;
         let selectedOptions;
 
-        if (some(nestedTogglesData.value, selectedOption)) { // Subtract
+        if (nestedTogglesData.singleSelection) {
+            selectedOptions = [selectedOption];
+        } else if (some(nestedTogglesData.value, selectedOption)) { // Subtract
             selectedOptions = filter(nestedTogglesData.value, (d) => (
                 d.value !== selectedOption.value
             ));
         } else { // Add
-            selectedOptions = sortBy([...nestedTogglesData.value, selectedOption], ['value']);
+            selectedOptions = sortBy([...(nestedTogglesData.value) || [], selectedOption], ['value']);
```

Comment:
Are the parentheses necessary here?
Or was the intent `...(nestedTogglesData.value || [])`?

Reply:
@groberts314 the component will crash if it has no value (`null` or `undefined`), it's helpful update.

Reply:
Sorry; I was asking if it could be simplified to this:
```JavaScript
selectedOptions = sortBy([...nestedTogglesData.value || [], selectedOption], ['value']);
```
(no parentheses around `nestedTogglesData.value` because they appear unnecessary, unless I'm missing something...)

Reply:
Fixed

Reply:
Thank you sir for the fix!  Apologies if the initial observation here was unclear, causing a lot of much back-and-forth.
Resolving.